Sources/TensorFlow/Operators/NN.swift

```diff
@@ -64,7 +64,7 @@ extension Padding {
   }
 
   @inlinable
-  internal var raw2: _Raw.Padding2 {
+  internal var raw2: _Raw.Padding1 {
```

Comment:
What's the story with `Padding2` vs `Padding1`, did these change because of code generation changes?

Reply:
The code generator would use a single number for deduping all enum types. This means that if you had two Padding and two Alignment or something it could show up like so:
Alignment
Alignment1
Padding
Padding2
This means that if another duplicate showed up somewhere, it could bump all the names. I changed it so that it just has a counter per base name instead of this.